src/stream/src/common/table/state_table.rs

```diff
@@ -682,41 +685,94 @@ where
             .await
             .map_err(Into::into)
     }
+}
+
+#[must_use]
+pub struct StateTablePostCommit<
```

Comment:
Can we add assertion for `self.inner.on_post_commit` when it gets `drop`ped?

Reply:
No. We cannot ensure that the `post_yield_barrier` can be called if the actor is aborted after yield barrier but before calling `post_yield_barrier`.